uvdat/core/tasks/networks.py

```diff
@@ -213,6 +213,35 @@ def geojson_from_network(dataset):
     return new_geodata.to_json()
 
 
+def vector_features_from_network(network):
```

Comment:
To clarify that this creates vector features and doesn't just return them:
```suggestion
def create_vector_features_from_network(network):
```

Reply:
Changed in a287c78